react-shopping-cart/src/pages/ProductList/ProductList.page.jsx

```diff
@@ -5,7 +5,7 @@ import ProductListBox from 'components/ProductListBox/ProductListBox.component';
 import ProductListContainer from 'components/ProductListContainer/ProductListContainer.component';
 import SkeletonItem from 'components/SkeletonItem/SkeletonItem.component';
 
-import useFetch from 'hooks/useFetch';
+import useLoadProducts from 'hooks/api/products/useLoadProducts';
 
 function LoadingSection() {
```

Comment:
```suggestion
export default function ProductList() {
  const { products, isLoading, loadProducts } = useLoadProducts();

  if (isLoading) return <LoadingSection />;

  return (
    <>
      <Header />
      <PageContainer>
        <ProductListContainer products={products} loadProducts={loadProducts} />
      </PageContainer>
    </>
  );
}

function LoadingSection() {
  return (
    <ProductListBox>
      {new Array(8).fill('').map((_, idx) => (
        <SkeletonItem key={idx} />
      ))}
    </ProductListBox>
  );
}
```

순서를 정리해본다면 이렇게도 할 수 있을것 같네요. 참고정도만 해주세요ㅎ

Reply:
ProductList부터 오는 것이 더 맞는 순서일 것 같네요!
수정했습니다! [3f98bfe](https://github.com/woowacourse/react-shopping-cart-prod/pull/76/commits/3f98bfe04afcd578dbd52cc2456c146c42c1f233)